### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.3)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#55859]
#### VFS team name
VA Online Scheduling (VAOS)

#### VFS product name
VA Online Scheduling

#### VFS feature name
Appointment List

#### Point of Contact/Reviewers
Allison Christman - @allison0034 - Design

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
Both the "Go to..." buttons are navigation.

### Link, screenshot or steps to recreate
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/42025-Allison-040703.png

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Navigation should be links and not buttons. Change to a link or action link

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by ldelacosta** [comment]
@shiragoodman - We reviewed this ticket and determined that VAOS does not own or manage this page/code. This ticket may need Public Website assistance to review. If you have any questions, please reach out. Thanks! cc: @JRRoof

**Comment by wesrowe** [comment]
@laflannery, can you please have a look and see how this sits in our a11y backlog? 

Context: This issue will be resolved by the transition of the functionality to MyHealtheVet on VA.gov. At that point there will be a redesign to a single CTA (link most likely).

**Comment by laflannery** [comment]
@wesrowe Do you know when that transition and redesign is planned for?

**Comment by wesrowe** [comment]
It's under way already, but not sure when their first launch is. The team responsible is in the ecosystem, I could find out.

**Comment by laflannery** [comment]
If it's underway already then I would say we hold off on this and just let it roll into the redesign.

**Comment by outerpress** [comment]
Adding context: the VAOS team is working on updating the widget that appears on the appointments static landing page. [Design spec](https://www.figma.com/file/iqjkZx2EbrZu6leL6YfN45/Cerner-Handoff?type=design&node-id=1631-22727&mode=design&t=ZGJAK8tsM4MsLfQJ-4).

[Dev ticket](https://github.com/department-of-veterans-affairs/vets-website/pull/24928)

**Comment by outerpress** [comment]
@ldelacosta this issue is resolved for the appts widget so I think you can close it.

<img width="1496" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN0NCQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--42ec5f2b254a0626a74e114c9eb3d18ae8752da3/Screenshot%202024-02-22%20at%209.48.19%E2%80%AFAM.png" alt="Screenshot 2024-02-22 at 9.48.19 AM.png" />
